metricbeat/module/ceph/_meta/Dockerfile

```diff
@@ -11,3 +8,8 @@ HEALTHCHECK --interval=1s --retries=300 \
         | jq .output.health.health_services[0].mons[0] \
         | grep health
 EXPOSE 5000
+
+ENV NETWORK_AUTO_DETECT 4
+ENV DEMO_DAEMONS osd,rest_api
```

Comment:
New image starts a UI in the same port as the rest_api, preventing the rest_api to work, I'm not sure what is the daemon starting the UI, but with this set of daemons the api starts correctly.